source/val/validate.cpp

```diff
@@ -286,34 +286,12 @@ spv_result_t ValidateBinaryUsingContextAndValidationState(
 
       if (auto error = IdPass(*vstate, inst)) return error;
     }
-
```

Comment:
I think line 245 and 246 can change to:

for (const auto& instruction : vstate->ordered_instructions()) {

since the pass that needed i is moved to the second batch.

Reply:
Will do